core/src/main/java/uk/gov/justice/services/core/json/FileBasedJsonSchemaValidator.java

```diff
@@ -28,6 +30,8 @@
     @Inject
     PayloadExtractor payloadExtractor;
 
+    DefaultJsonValidationLoggerHelper defaultJsonValidationLoggerHelper = new DefaultJsonValidationLoggerHelper();
```

Comment:
This can be injected as a JsonValidationLoggerHelper

Reply:
JsonValidationLoggerHelper does not have knowledge of ValidationExceptions so cannot work here. I have now injected the DefaultJsonValidationLoggerHelper instead of creating.

Reply:
Have tried to inject this but there is a brittle dependency on JsonSchemaValidatingMockTest.